src/features.rs

```diff
@@ -57,7 +58,24 @@ pub fn list_languages(assets: &HighlightingAssets, term_width: usize) {
 }
 
 pub fn print_files(assets: &HighlightingAssets, config: &Config) -> Result<bool> {
-    let theme = assets.get_theme(config.theme.unwrap_or("Default"))?;
+    let theme = match config.theme {
```

Comment:
This whole logic should probably be moved to the `App::config` function in `app.rs`. Once `print_files` is called, everything should already be configured properly (and `config` should be ready to use). 

Also, instead of using `match` statements on `Option`, I think we should be able to use combinator functions from [`std::option`](https://doc.rust-lang.org/std/option/enum.Option.html). For example:

```rust
env::var("BAT_THEME").or(self.matches.value_of("theme")).unwrap_or("Default")
```

Concerning error handling, I'd be okay with always using `?` (fatal error).